action.yml

```diff
@@ -94,9 +97,33 @@ runs:
     #     # TODO: implement the config via the Python script below later:
     #     # python3 ${ACTION_PATH}/special_config.py -b "${{ inputs.beamline-acronym }}"
 
-    - name: Beamline Specific Configurations
-      env:
-        BEAMLINE_ACRONYM: ${{ inputs.beamline-acronym }}
+    - name: Beamline-specific env vars
+      shell: bash -leo pipefail {0}
+      run: |
+        echo "::group::Setting up beamline-specific environment variables"
+        beamline_acronym="${{ inputs.beamline-acronym }}"
+        export BEAMLINE_ACRONYM="${beamline_acronym,,}"  # lowercase it
+        echo "BEAMLINE_ACRONYM=${BEAMLINE_ACRONYM}" >> $GITHUB_ENV
+
+        export BEAMLINE_ACRONYM_UPPER="${BEAMLINE_ACRONYM^^}"  # uppercase it
+        echo "BEAMLINE_ACRONYM_UPPER=${BEAMLINE_ACRONYM_UPPER}" >> $GITHUB_ENV
```

Comment:
Nitpick: acronyms are typically uppercase by default, so I think we should have `BEAMLINE_ACRONYM = TLA` and `BEAMLINE_ACRONYM_LOWER = tla`